### PR TITLE
EL-429: Subject Matter of Dispute for property

### PR DESCRIPTION
**Pull request by starswan**
https://dsdmoj.atlassian.net/browse/EL-429

Add SMOD checkboxes to property and second property screens

### CODE REVIEW

File: app/services/submit_assets_service.rb [hunk 1]
```diff
@@ -27,8 +28,23 @@ def call(cfe_estimate_id, cfe_session_data)
...
     end
 
-    cfe_connection.create_properties(cfe_estimate_id, main_home, second_property) if main_home.present? || second_property.present?
+    create_properties(cfe_estimate_id, main_home, second_property) if main_home.present? || second_property.present?
+  end
+
+private
+
+  def create_properties(assessment_id, main_property, second_property)
+    main_home = main_property ||
```
Comment (patrick-laa) [inline app/services/submit_assets_service.rb:40]: :+1: to moving this logic

**Review by patrick-laa — APPROVED** [review]
A suggestion for a simplification; otherwise all good.

File: app/forms/property_entry_form.rb [hunk 2]
```diff
@@ -18,4 +18,10 @@ class PropertyEntryForm
   validates :percentage_owned,
             numericality: { greater_than: 0, only_integer: true, less_than_or_equal_to: 100, allow_nil: true, message: :within_range },
             presence: true
+
+  attribute :house_in_dispute, array: true, default: []
```
[the remark this answers is not in the document]
Comment (starswan) [reply app/forms/property_entry_form.rb:22]: Great thanks - originally used a boolean, but couldn't work out how to fit it in with the form submit. Think multiple: false is what I need

File: config/locales/en.yml [hunk 3]
```diff
@@ -155,6 +155,10 @@ en:
     date: Date
     related_content_title: Related content
     not_applicable: Not applicable
+    dispute: This asset is a subject matter of dispute
```
Comment (willc-work) [inline config/locales/en.yml:158]: I think this should say "This asset is the subject matter of dispute" or "This asset is subject matter of dispute". The inclusion of the 'a' scans funny to me. And in the guidance it is referred to using either 'the' or no article at all

**Pull request closed without merging**